sphinx_needs_enterprise/services/codebeamer.py

```diff
@@ -75,18 +76,20 @@ def request(self, options=None):
                 "descFormat": "HTML",
             },
         }
-
-        answer = self._send_request(request_params)
+        time.sleep(0.1)
+        answer = self._send_request(request_params, params["cert_abspath"])
         data = answer.json()["items"]
         for datum in data:
+            time.sleep(0.1)
```

Comment:
codebeamer has been complaining about receiving requests too quickly. add 100 ms delay here

Reply:
Would like to not hard-code this here, as there are codebeamer instance which don't have a problem with it.

Maybe we can have a config variable, which is by default 0 and can be changed in conf.py